.github/workflows/old.WCYCL1850.ne4_oQU240.yaml

```diff
@@ -0,0 +1,36 @@
+name: WCYCL1850.ne4_oQU240
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+ 
+  workflow_dispatch:
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    container: 
+      image: ghcr.io/mahf708/e3sm-imgs@sha256:2657196ea9eec7dbd04f7da61f4e7d5e566d4b501dff5881f0cb5125ba031158
```

Comment:
I'm assuming this container has all TPLs already installed? If so, what's the point of the apg-get calls below?